docs/user-guide/rules.md

```diff
@@ -1,6 +1,8 @@
 # Rules
 
-Every rule is standalone and turned off by default. None of the rules have default values for their options.
+There are over a hundred rules built into stylelint. They all focus on *standard css* and aim to be valuable to the majority of users. In addition to these rules there are [plugins](/docs/user-guide/plugins.md), which are rules built by the community that support methodologies, toolsets, *non-standard* CSS features, or very specific use cases.
+
+Every rule and plugin is standalone and turned off by default. None of the rules or plugins have default values for their options.
 
 ## List of rules
```

Comment:
Let's throw in another sentence at the end of "Rules" or beginning of "List of rules" saying: "Don't forget to look at the list of plugins for more ways to lint."